### PR TITLE
(DOCSP-30466) Adds RNs for 1.7.2

### PR DESCRIPTION
**Pull request by erabil-mdb**
[JIRA](https://jira.mongodb.org/browse/DOCSP-30466)
[BUILD](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6482142667a273b8169095d2) (includes [expected errors](https://mongodb.slack.com/archives/CGX40QH7Z/p1679333780930919?thread_ts=1679333691.066079&cid=CGX40QH7Z))
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-30466/atlas-cli-changelog/#atlas-cli-1.7.2)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -14,6 +14,21 @@
...
    :class: singlecol
 
+.. _atlas-cli-1.7.2:
+
+{+atlas-cli+} 1.7.2
+-------------------
+
+Released 7 June 2023
+
+- Fixes a bug that caused the :ref:`atlas-clusters-search-indexes-create` command to
```
Comment (wtrocki) [inline source/atlas-cli-changelog.txt:24]:
```suggestion
- Fixes an issue that caused the :ref:`atlas-clusters-search-indexes-create` command to 
```

**Review by sarahsimpers — APPROVED** [review]
LGTM, thanks!

File: source/atlas-cli-changelog.txt (hunk 1 above)
Comment (wtrocki) [reply source/atlas-cli-changelog.txt:24]: To align with second point

Comment (erabil-mdb) [reply source/atlas-cli-changelog.txt]: Good point! Changed.

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -14,6 +14,21 @@
...
+{+atlas-cli+} 1.7.2
+-------------------
+
+Released 7 June 2023
+
+- Fixes a bug that caused the :ref:`atlas-clusters-search-indexes-create` command to 
+  incorrectly handle a multikey index.
+- Fixes an issue with the :ref:`atlas-performanceAdvisor-slowQueryLogs` command where the
+  output didn't display correctly in tabular form.
+- Adds the ``--tag`` flag to the commands :ref:`atlas-clusters-create`, :ref:`atlas-clusters-update`,
```
[the remark this answers is not in the document]
Comment (erabil-mdb) [reply source/atlas-cli-changelog.txt]: Removed!

[the remark this answers is not in the document]
Comment (sarahsimpers) [reply source/atlas-cli-changelog.txt]: Hi @wtrocki Was this recently reverted? I see it appears on the docs page in staging for atlas clusters create so wondering if that needs to change too: https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-30466/command/atlas-clusters-create/#options

**Review by wtrocki — APPROVED** [review]
Awesome work. 
Thank you @erabil-mdb

File: source/atlas-cli-changelog.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (wtrocki) [reply source/atlas-cli-changelog.txt]: Change is waiting for API to release. Then it be included in the next version.
This is tradef us using atlas-cli-next release. If there is hot fix in the mean time while we wait for feature release we going to have changelog with invalid entries. It should not happen that offten

**Pull request merged**